config.json

```diff
@@ -588,280 +588,252 @@
         "difficulty": 1
       },
       {
-        "slug": "rna-transcription",
-        "name": "RNA Transcription",
-        "uuid": "a4629a60-752f-4234-99f1-6b3ac0b4ba18",
+        "slug": "resistor-color",
+        "name": "Resistor Color",
+        "uuid": "c1532449-bf9e-446a-8e8f-97a6c19608e5",
         "prerequisites": [
-          "charlists",
-          "pattern-matching",
-          "multiple-clause-functions",
-          "case",
-          "enum"
+          "atoms",
+          "maps"
         ],
         "practices": [
-          "charlists"
+          "atoms",
+          "maps"
         ],
-        "difficulty": 2
+        "difficulty": 1
       },
       {
-        "slug": "word-count",
-        "name": "Word Count",
-        "uuid": "4a24ba2f-ae92-4095-be53-64bc881422ea",
+        "slug": "simple-linked-list",
```

Comment:
`simple-linked-list` is difficulty 1??
No way, that exercise is very abstract, a solid 5 in my book, you need to know what a linked list even is. I checked community solutions on the site and a solid proportion of answers are all wrong, using `Tuple.insert_at` or even `List` functions. A difficulty 1 should have 99% correct answers on the first submission.

Reply:
Totally agree with you, there's no way it's a 1, which would mean "it's as easy as `two-fer` and `resistor-color`, but easier than `resisitor-color-duo`" 😂. Maybe the original author thought it should be solved with the built in linked lists.

That's my main worry about this ordering, that we got the difficulties wrong. `markdown` is a 5... I think I would give `simple-linked-list` a 4 and `markdown` a 6 or 7?

Another very badly scored exercise is `bank-account`. It has a 7, but it's a very straight-forward use of `Agent` or `GenServer`, should be a 3-4?

Reply:
Yes that seems better. 
I have solved a bunch, but not all, so it's hard to judge them all :)

Can we have stats on how many have submitted solutions for each exercise? That could possibly give us a general idea for difficulties.

Reply:
We could ask Jeremy to give us those stats, but I fear they would be skewed by the current unlocking order and core exercise. E.g. everyone will have solved "rna transcription" because you need it do unlock anything else, but few people might attempt "bowling" because you need to do "markdown" and "zipper" first. If you're in mentored mode, not interdependent mode.

Reply:
Of course it will be skewed, but it might be useful for non-track exercises. Or at least interesting.

Reply:
Let's talk about that on Slack 👍